GUI/MainChangeset.cs

```diff
@@ -22,22 +22,59 @@ public void UpdateChangesDialog(List<KeyValuePair<GUIMod, GUIModChangeType>> cha
                 return;
             }
 
+            m_Changeset = new List<ModChange>();
+            m_Changeset.AddRange(changeset.Where(change => change.ChangeType == GUIModChangeType.Remove));
+            m_Changeset.AddRange(changeset.Where(change => change.ChangeType == GUIModChangeType.Update));
+
+            IEnumerable<ModChange> left = changeset.Where(change => change.ChangeType == GUIModChangeType.Install);
```

Comment:
`left` being everything which is left after we've excluded removes and updates? I feel that would be more clear if our linq-code was everything in `changeset` excluding everything in `m_Changeset`, which means `left` is forward-consistent even if we have new changetypes in the future (eg: a "reconfigure" changetype).

Otherwise I'd be more inclined to call this `install_list` or similar. :)